pkg/kube/client.go

```diff
@@ -929,6 +929,24 @@ func (c *Client) watchPodUntilComplete(timeout time.Duration, info *resource.Inf
 	return err
 }
 
+// GetPodLogs takes pod name and namespace and returns the current logs (streaming is NOT enabled).
+func (c *Client) GetPodLogs(name, ns string) (string, error) {
```

Comment:
Any way we can return an `io.ReadCloser` instead of a `string` here and let `GetLogs` handle the stream? That way, the consumer (`GetLogs`) can determine what to do with the buffer instead of always reading it into memory. For example, what if the consumer just wanted to open the stream to test that it was possible (like in a test case)?